packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -187,12 +229,36 @@ export class CommonFrontendContribution implements MenuContribution, CommandCont
         commandRegistry.registerCommand(CommonCommands.REPLACE);
 
         commandRegistry.registerCommand(CommonCommands.NEXT_TAB, {
-            isEnabled: () => this.app.shell.hasSelectedTab(),
-            execute: () => this.app.shell.activateNextTab()
+            isEnabled: () => this.shell.hasSelectedTab(),
+            execute: () => this.shell.activateNextTab()
         });
         commandRegistry.registerCommand(CommonCommands.PREVIOUS_TAB, {
-            isEnabled: () => this.app.shell.hasSelectedTab(),
-            execute: () => this.app.shell.activatePreviousTab()
+            isEnabled: () => this.shell.hasSelectedTab(),
+            execute: () => this.shell.activatePreviousTab()
+        });
+
+        commandRegistry.registerCommand(CommonCommands.CLOSE_TAB);
```

Comment:
just FYI, you can do:
```ts
commandRegistry.registerCommand(CommonCommands.CLOSE_TAB, {
    execute: () => this.shell.closeTab()
});
```

Reply:
Right I forgot about that again.. fixing.